python/perftest/templates/_macros.j2

```diff
@@ -68,11 +78,21 @@ volumeMounts:
 name: wait-for-{{ component }}-port-{{ port }}
 image: {{ settings.discovery_container_image }}
 imagePullPolicy: {{ benchmark.spec.image_pull_policy | default(settings.default_image_pull_policy) }}
+securityContext:
+  privileged: true
```

Comment:
I'd still like to understand why we need to do this, and the symlinking below, since it was working without this before. I don't like having to use a privileged container - if we do need to do something, can we do it by granting a specific capability?

Reply:
I don't have a good answer for the underlying issue, but do at least have a workaround that doesn't require `securityContext.privileged: true`, which was admittedly a grim hack.